src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.module.css

```diff
@@ -35,11 +35,11 @@
 .variantType {
   font-size: 12px;
   font-weight: var(--font-weight-light);
-  margin-left: 0.6rem;
+  margin-left: 12px;
 }
 
 .labelTextStrong {
-  font-size: 12px;
+  font-size: 13px;
   font-weight: var(--font-weight-bold);
 }
 
```

Comment:
Here's what this whole review started with, isn't it? Compare this to the styles in the XD

<img width="245" alt="image" src="https://github.com/Ensembl/ensembl-client/assets/6834224/24858d48-6b01-440a-94a3-91cbee012205">